### ISSUE TITLE
Google Maps API Geschäfts-Konto anlegen

### ISSUE THREAD

**Issue by lehoffma** [lehoffma/memo#78]
Da die Google Maps Api ohne Kreditkarten-Identifikation nach 1000 Anfragen pro Tag gesperrt wird, müssen wir da ein "Geschäftskonto" anlegen (siehe [hier](https://developers.google.com/places/web-service/usage#identitat_durch_aktivieren_der_abrechnungsfunktion_uberprufen)). 

Beim Anlegen des Kontos wollen die einige Kontoinfos und eine Kreditkarte haben (nur zur Verifizierung bzw. wird nur zur Abbuchung benutzt, falls wir seperat den Premium-Plan aktivieren). Soll ich als Unternehmensname dann Meilenwölfe e.V. schreiben und bei Addresse etc. die Daten ausm Meilenwölfe Impressum nehmen? Und welche Kreditkarte? Ich hätte jetzt einfach meine genommen, aber ich dachte mir dass ich euch vorher lieber mal frage.

**Comment by sinnloserName4** [comment]
Ich organisiere die Daten für die Meilenwölfe Kreditkarte. Da wir im Entwicklungsbetrieb ja bestimmt nicht über 1000 am Tag kommen, hat das aber bestimmt noch 2 Wochen zeit, oder?

**Comment by lehoffma** [comment]
Was meinst du warum ich den Thread erstellt hab 😄 
Ich hab an der Tour-Erstellen Seite gearbeitet und da bin ich nach n paar Stunden an das Limit gekommen.

**Comment by sinnloserName4** [comment]
kann man die Kreditkartendaten noch ändern? Viel früher komme ich nicht an die Daten der Meilenwölfe.
Als Adresse kannst du bei Firma Meilenwölfe e.V. eintragen. Als person wer auch immer die Kreditkarte hat (also temporär z.B. du) und als Adresse entweder deine oder

Meilenwölfe e.V.
c/o Sören Henke
Alte Bahnhofstraße 37
38470 Parsau, OT Croya

**Comment by lehoffma** [comment]
Nee, die Kreditkarte wird ja zur Identifikation benutzt, ohne ändert sich da wenig. Aber dann müssen wir halt bis dahin erstmal so auskommen. Am nächsten Tag is das Kontigent ja wieder zurückgesetzt.

**Comment by lehoffma** [comment]
Die API Kosten Berechnung haben sich komplett geändert und beim kurz drüber gucken scheint das ganze ein wenig unübersichtlicher geworden zu sein. 

[Hier](https://developers.google.com/maps/billing/understanding-cost-of-use) steht n bisschen mehr. Da müsste man sich aber noch mal genauer mit beschäftigen, um zu schauen, ob das auf uns irgendeine Auswirkung hat.

**Comment by sinnloserName4** [comment]
Welche APIs benutzen wir denn? Würde jetzt wo wir offiziell live gegangen sind dann ein neues Konto für die Meilenwölfe mit Kreditkarte etc erstellen und dann die API-Keys austauschen. Letztlich können wir eh nicht ändern wenn es jetzt mehr kostet
